### PR TITLE
Adding coverage to digest for parser

### PR DESCRIPTION
**Pull request by sg3-141-592**
This additional fuzzer `fuzz_params` calls the `digest` utility mocking the stdin and parser args inputs. This is closer to how this utility is used normally and gives additional coverage.

This has shook out an issue pretty quickly https://github.com/bmc/digest/issues/4 .

Another issue, the introspector doesn't seem to be able to measure any coverage on digest at all.

image

### CODE REVIEW

**Comment by DavidKorczynski** [comment]
> Another issue, the introspector doesn't seem to be able to measure any coverage on digest at all.

Can you please post stuff like this on https://github.com/ossf/fuzz-introspector/issues rather than here, as it's just going to be lost down the line if we have these things in PRs rather than issues on right repo.

File: projects/digest/fuzz_params.py
Comment (DavidKorczynski) [inline projects/digest/fuzz_params.py:40]: No need for `enable_python_coverage`

**Review by DavidKorczynski — CHANGES_REQUESTED** [review]

File: projects/digest/fuzz_params.py
Comment (sg3-141-592) [reply projects/digest/fuzz_params.py]: Removed `enable_python_coverage`

**Comment by sg3-141-592** [comment]
Understood on putting the missing coverage onto issues for introspector. This library is entirely in an `__init__.py` file so it's likely fixed by https://github.com/ossf/fuzz-introspector/pull/1005 . I'll wait for the introspector to be bumped in oss-fuzz to confirm.

**Review by DavidKorczynski — CHANGES_REQUESTED** [review]
I don't think this fuzzer is really needed as it's kinda superficial. The library is so small https://github.com/bmc/digest/blob/master/digest/__init__.py and the logic is already covered -- don't see any reason for calling by way of `main` from a code analysis perspective

**Comment by sg3-141-592** [comment]
Sure, I'm happy to leave it up to you what you do with it.

Hitting the `main` using mocked `stdin` and `argv` means you get more coverage than hitting `digest` directly. But you're right this is very superficial, it's just a lightweight wrapper for `hashlib` so fuzzings not going test much.

**Comment by Alan32Liu** [comment]
Hope you won't mind me converting it to a draft for now until we have a decision (merge or close): )

**Comment by DavidKorczynski** [comment]
Closing this due to the reasons mentioned above.

**Pull request closed without merging**
